ruby/basics/variables_strings_and_numbers.md

```diff
@@ -1,119 +1,371 @@
 
 # Variables, Strings and Numbers
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+Since you've already done a decent chunk of Ruby in the prep course, this should be a healthy refresher of what you've already learned with Variables, Strings and Numbers.
+
+This lesson will build on your knowledge and go much deeper and further than you went before.
+
+Just like you can't build a castle on a weak foundation, neither can you write a great program without having strong knowledge of the more basic data types. If you want to write the next [Facebook](www.facebook.com) or [LingsCars](www.lingscars.com) you'll need a fundamental knowledge of basic data types and what you can do with them.
+
+Variables, Strings and Numbers can be a little unappreciated next to much cooler Ruby features like Blocks and Hashes but they are so fundamental to Programming that you should know them inside out and what you can do with them before you can really master more advanced features.
+
+We'll start with basic concepts before diving into more advanced areas for each topic so that by the end you'll be comfortable using and manipulating Strings and Numbers and storing them in an appropriately named Variable.
+
+Let's get started. Who's excited??????
 
 ## Learning outcomes
 *Look through these now and then use them to test yourself after doing the assignment*
 
-what the student is expected to know or be able to do by the end of this lesson
+* Variables
+  * What is a variable?
+  * How do you assign a variable?
+  * what does the `+=` operator do?
+  * What are the variable naming conventions?
+
+* Strings
+  * What is a string?
+  * What's the difference between single and double quotes?
+  * What is string interpolation?
+  * How do you concatenate strings?
+  * What method would you use to change all the characters in your string to upper case?
+  * What method would you use to _split_ up strings into arrays?
+  * What are escape characters?
+  * How do you access a specific character or substring?
+  * How do you make other data types into strings?
+
+* Numbers
+  * What are the basic arithmetic operators you can use with numbers?
+  * What's the difference between an Integer and a Float?
+  * What method would you use to convert a float to an integer?
+  * What method would you use to convert a integer to a float?
 
-* Learning outcome 1
-* Learning outcome 2
-* Learning outcome 3
 
 ## Variables
-You should know what variables are from the prep course and know how to assign values to a variable name
-.
+You can think of Variables as those tupperware containers your mum keeps in the cupboard (that never have matching lids...). You can use those containers to store something in them until you need whatever is in it. In programming a variable is simply a place to store data. To be more specific, variables hold a reference to a specific object in computer memory. Each object will refer to a certain data type.
```

Comment:
I think the Tupperware analogy works as an initial introduction to variables. 

Perhaps, changing "To be more specific" into "To be more technical" allows for a more logical flow into a more nuanced understanding of them. Psychologically, I think it would allow people to think "Oh, technical stuff, I'm not ready for that yet" and they can just skip right through.